beta-src/src/utils/api/index.ts

```diff
@@ -26,10 +26,10 @@ export const getGameApiRequest = (
   route: ApiRoute,
   queryParams: QueryParams,
 ): Promise<AxiosResponse> =>
-  api.get(`?route=${route}&${buildQueryString(queryParams)}`);
+  api.get(`api.php?route=${route}&${buildQueryString(queryParams)}`);
```

Comment:
shouldn't these be `/api.php` the env variable is `REACT_APP_WD_BASE_URL=http://localhost` without the slash at the end.

Reply:
No because axios automatically adds the trailing slash